custom-webpack/universal-starter-master/src/app/app.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { TransferState } from '../modules/transfer-state/transfer-state';
+import { REQUEST } from '@nguniversal/express-engine/tokens';
+
+@Component({
+  selector: 'demo-app',
+  template: `
+    <h1>Universal Demo using Angular</h1>
+    <a routerLink="/">Home</a>
+    <a routerLink="/lazy">Lazy</a>
+    <router-outlet></router-outlet>
+  `,
+  styles: [
+    `h1 {
+      color: green;
+    }`
+  ]
+})
+export class AppComponent implements OnInit {
+  constructor(private cache: TransferState) { }
+
+  ngOnInit() {
+    // This is an example
```

Comment:
this is an example of storing a value on the server to be sent to the client

Reply:
I havn't changed anything here, this should be the same as master. So I could make the change but I think this PR should be just about brining the CLI in rather than fixing other issues